test/JDBC/input/views/sys-computed_columns-vu-verify.sql

```diff
@@ -3,3 +3,21 @@ GO
 
 SELECT COUNT(*) FROM sys.all_columns WHERE object_id = object_id('sys.computed_columns');
 GO
+
+Select definition FROM sys.computed_columns WHERE name='scc_first_number';
```

Comment:
Modify the test like this to avoid so many lines:

```
SELECT name, definition
FROM sys. computed_columns
WHERE name in ('scc_first_number', 'scc_second_number', 'scc_multiplied')
ORDER BY name

Reply:
Changes done in next commit.